src/main/java/segelzwerg/sporttooolbox/iunits/distance/Nautical.java

```diff
@@ -86,6 +84,16 @@ public Kilometer toKilometer() {
         return new Kilometer(getNautical() * NAUTICAL_TO_KM);
     }
 
+    @Override
+    public Miles toMiles() {
+        return new Miles(getNautical() * NAUTICAL_TO_KM / MILES_TO_KM);
```

Comment:
```suggestion
        return new Miles(getNautical() * NAUTICAL_TO_MILES);
```
shouldn't we use the one constant instead of two here?